WifiART/src/userinterface/ArtMvcView.java

```diff
@@ -248,69 +321,195 @@ public SubelementName getSelectedTab() {
+     *
+     * @param listener the listener for the parameter.
+     */
     public void addOutputFileNameListener(ActionListener listener) {
         outputFileNameField.addActionListener(listener);
     }
+
+    /**
+     * Add a listener for the output file directory.
+     *
+     * @param listener the listener for the parameter.
+     */
     public void addOutputDirListener(ActionListener listener) {
         outputDirField.addActionListener(listener);
     }
+
+    /**
+     * Add a listener for whether or not the buffer display should be in a readable form.
+     *
+     * @param listener the listener for the parameter.
+     */
     public void addReadableListener(ActionListener listener) {
         readableBufferCheckbox.addActionListener(listener);
     }
+
+    /**
+     * Add a listener for whether or not the newer Android version is being used.
+     *
+     * @param listener the listener for the parameter.
+     */
     public void addAndroidVersionListener(ActionListener listener) {
         androidVersionCheckbox.addActionListener(listener);
     }
+
+    /**
+     * Add a listener for generating the buffer.
+     *
+     * @param listener the listener for generating the buffer.
+     */
     public void addGenerateBufferListener(ActionListener listener) {
         generateButton.addActionListener(listener);
     }
+
+    /**
+     * Add a listener that is triggered by switching tabs.
+     *
+     * @param listener the listener to be triggered when switching tabs.
+     */
     public void addTabbedPaneListener(ChangeListener listener) {
         tabbedPanel.addChangeListener(listener);
     }
 
     // Getters for the sub-views
+
+    /**
+     * Gets the view for the LCI subelement.
+     *
+     * @return the LCI subelement view.
+     */
     public LciView getLciView() {
         return lciView;
     }
+
+    /**
+     * Gets the view for the Z subelement.
+     *
+     * @return the Z subelement view.
+     */
     public ZView getZView() {
         return zView;
     }
+
+    /**
+     * Gets the view for the Usage Rules/Policy subelement.
+     *
+     * @return the Usage Rules/Policy subelement view.
+     */
     public UsageView getUsageView() {
         return usageView;
     }
+
+    /**
+     * Gets the view for the BSSID List subelement.
+     *
+     * @return the BSSID List subelement view.
+     */
     public BssidView getBssidView() {
         return bssidView;
     }
+
+    /**
+     * Gets the view for the Location Civic subelement.
+     *
+     * @return the Location Civic subelement view.
+     */
     public LcrView getLcrView() {
         return lcrView;
     }
+
+    /**
+     * Gets the view for the Map Image subelement.
+     *
+     * @return the Map Image subelement view.
+     */
     public MapView getMapView() {
         return mapView;
     }
 
+    /**
+     * Displays the buffer in the window.
+     *
+     * @param lciBuffer the list of included LCI subelement buffers
```

Comment:
If lciBuffer is a list, you should include it in the name, and then you don't need to include it in the description.

Reply:
Good call. This parameter originally referred to something different, but I didn't change the name when I switched its function.